### PR TITLE
Bugfix/minsize refresh

### PR DESCRIPTION
**Pull request by toaster**
This extends the fix for #264.

### CODE REVIEW

File: internal/driver/gl/canvas.go [hunk 1]
```diff
@@ -191,12 +192,29 @@ func (c *glCanvas) ensureMinSize() bool {
 	if c.Content() == nil {
 		return false
 	}
-	ensureMinSize := func(obj fyne.CanvasObject, _ fyne.Position, _ bool) {
-		if obj.Visible() {
-			expectedSize := obj.MinSize().Union(obj.Size())
-			if expectedSize != obj.Size() {
+	var objToBeLayouted fyne.CanvasObject
```
Comment (andydotxyz) [inline internal/driver/gl/canvas.go]: Layouted isn't a word - how about objToLayout or layoutObject?

File: internal/driver/gl/canvas_test.go [hunk 2]
```diff
@@ -77,3 +76,86 @@ func Test_glCanvas_SetContent(t *testing.T) {
 		})
 	}
 }
+
+func Test_glCanvas_GrowingMinSize(t *testing.T) {
```
Comment (andydotxyz) [inline internal/driver/gl/canvas_test.go]: Is it possible to shrink this test down or split it into multiple?
It's hard to read as is and not all items are checked in the assert. This is an open question and if this is the minimum required that's fine.

File: internal/driver/gl/canvas.go (hunk 1 above)
Comment (toaster) [reply internal/driver/gl/canvas.go]: Yeah, thanks, I'll go fot `objToLayout`.

File: internal/driver/gl/canvas_test.go (hunk 2 above)
Comment (toaster) [reply internal/driver/gl/canvas_test.go]: Yeah, I could split it up.

**Comment by toaster** [comment]
implemented feedback

**Pull request merged**